package.json

```diff
@@ -43,7 +43,7 @@
     "babel-core": "^6.22.1",
     "babel-loader": "^6.2.10",
```

Comment:
Resuming work on this. Yes, I'm upgrading everything. babel-loader 7 is now stable fortunately.